recipes/coverage/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "3.4.2" %}
```

Comment:
my bad, it was a leftover copy from a previous recipe submitted. supposed to be version 4.2